src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -554,6 +562,10 @@ private function formatControllerLink($controller, string $anchorText, callable
                 $r = new \ReflectionFunction($controller);
             }
         } catch (\ReflectionException $e) {
+            if (!\is_string($controller)) {
```

Comment:
This is the fix.

Reply:
I'm wondering if this shouldn't be replaced by 
```php
if (\is_array($controller)) {
	$controller = implode('::', $controller);
}
```
I feel like this might allow removing the added patch above, and follow the existing code path below that will fetch the kernel correctly.

Reply:
You are right! Great idea once again :smile: